pkg/kubelet/kubelet.go

```diff
@@ -1773,6 +1772,8 @@ func (kl *Kubelet) syncLoopIteration(configCh <-chan kubetypes.PodUpdate, handle
 
 		switch u.Op {
 		case kubetypes.ADD:
+			// Perform limits check for resolv.conf
+			kl.checkLimitsForResolvConf()
```

Comment:
My vision of when we should logging and eventing (https://github.com/kubernetes/kubernetes/pull/29666#issuecomment-247620361) was based on one of last comment in discussion https://github.com/kubernetes/kubernetes/pull/29666#issuecomment-240281656:

>A goroutine that monitors the flag-configured resolv.conf for > 3 search
paths and logs an event against the node is uncontroversial.

Monitoring of resolv.conf search line fits limits was an alternative to failure mode (unhealth state for node) which @vishh proposed and he agreed to do in more "safe" way. And kl.checkLimitsForResolvConf() checks only limits, not duplicates. 

So I supposed we'd like to have two type events per node and per pod's, and alarming logs for cluster admins on pod's creation if limits've been exceeded and periodically alarms on node's resolv.conf limits exceeded.